example/client/flex_await/Jamfile

```diff
@@ -8,11 +8,13 @@
 #
 
 using openssl ;
+import ac ;
 
 project
     : requirements
       $(c11-requires)
       <library>/boost/http_proto//boost_http_proto
+      [ ac.check-library /boost/http_proto//boost_http_proto_zlib : <library>/boost/http_proto//boost_http_proto_zlib : ]
```

Comment:
Don't you need to also conditionally compile the call to install the service?

Reply:
I used `BOOST_HTTP_PROTO_HAS_ZLIB` for that:
https://github.com/cppalliance/http_io/blob/24f1d92398e372bb5ed350ced7a668c03dafd8a1/example/client/flex_await/main.cpp#L334-L338

Reply:
Who sets `BOOST_HTTP_PROTO_HAS_ZLIB ` ?

Reply:
Build system. Yeah it doesn't look a good idea for an example.